docusaurus/docs/Android/03-ui/04-components/01-channel-list.md

```diff
@@ -67,17 +126,21 @@ TransformStyle.channelListStyleTransformer = StyleTransformer { defaultStyle ->
     )
 }
 ```
+
+These changes should have the following results:
```

Comment:
I'm not sure if I will keep this, but I am playing around with moving the before and after shots to after the instructions. I can see arguments for both orders, but this seems like it works a bit better by putting the code samples and actionable instructions up front.